README.md

```diff
@@ -61,7 +61,8 @@ ktlint {
 
 This plugin adds two tasks to every source set: `ktlint[source set name]Check` and `ktlint[source set name]Format`.
 Additionally, a simple `ktlintCheck` task has also been added that checks all of the source sets for that project.
-Similarly, a `ktlintFormat` task has been added that formats all of the source sets.
+Similarly, a `ktlintFormat` task has been added that formats all of the source sets. Also same meta tasks are added
+to the root project for convenience - they will call related tasks in subprojects. 
```

Comment:
True that user can see tasks from subprojects as well.

Previously when user applied plugin to the root project that contains subprojects - no ktlint tasks are added, but now user can do that - plugin will add tasks for subprojects and add them as a dependency in the root project meta task.

Will make description more understandable.